isis/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@
 
 # Specify the required version of CMake.  If your machine does not
 #  have this, it should be easy to build from https://cmake.org/download/
-cmake_minimum_required(VERSION 3.4)
+cmake_minimum_required(VERSION 3.10)
```

Comment:
Do we need to update anything in the system / default anaconda library to support this?

Reply:
The new environment files should be used to update that anaconda environments. We should be able to do `conda env update -f environment.yml` and it will update the environment.

Reply:
:+1: Yes -- this isn't a barrier to merging it in so much as a "when this gets merged in, what else may need to happen?"

Reply:
We also need to add the submodule initialization to the build/clone instructions and the build scripts.

Reply:
@jessemapel :+1: Good point!